compaction/compaction_descriptor.hh

```diff
@@ -72,6 +72,9 @@ public:
             only, // scrub only quarantined sstables
         };
         quarantine_mode quarantine_operation_mode = quarantine_mode::include;
+
+        // Should scrub::mode::validate move invalid sstables into qurantine
+        bool validate_qurantines_sstables = true;
```

Comment:
nit: fix typo: s/validate_qurantines_sstables/validate_quarantine_sstables/

I'd also name in a name that doesn't preclude other scrub types, since I can see a use case for quarantining also in ABORT or SKIP modes.

Reply:
Also how about defining a bool_class for that so the caller site would be more self-descriptive?

Reply:
Using `qurantines` instead of `qurantine` was deliberate, I wanted a verb instead of a noun.

Reply:
I think Benny was referring to missing 'a' before 'r'.